testsuite/features/build_validation/add_custom_repositories/add_ceos8_repositories.feature

```diff
@@ -15,7 +15,7 @@ Feature: Adding the CentOS 8 distribution custom repositories
     And I follow "Create Channel"
     When I enter "Custom Channel for CentOS 8 DVD" as "Channel Name"
     And I enter "centos-8-iso" as "Channel Label"
-    And I select the parent channel for the "ceos8_minion" from "Parent Channel"
```

Comment:
I prefer that we keep this step definition, and we use the value of the parent channel for that minion, stored in the constant. As we do in other sections inside BV testsuite.

Reply:
No. Problem is that it is **not** the parent channel for that minion, due to the CLM filters for appstream channels.

It therefore has to be hardcoded.

Reply:
After our discussion, I will add explanations at the beginning of feature. It is indeed clear as mud.